adapters/jsonLogger/adapter.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jsonLogger
```

Comment:
I changed it from stdoutLogger to jsonLogger to reflect that conceivably we could make it configurable to accept any io.Writer (not necessarily just stdio). Ultimately I don't have any strong feelings.

Maybe localLogger?